.github/actions/spell-check/dictionary/names.txt

```diff
@@ -1,57 +1,59 @@
-Anup
-austdi
-Ballmer
-bhoj
-Bhojwani
-dhowett
-Diviness
-dsafa
-duhowett
-ethanschoonover
-Firefox
-Gatta
-Grie
-Griese
-Hernan
-Howett
-Illhardt
-jantari
-jerrysh
-Kaiyu
-kimwalisch
-KMehrain
-Kourosh
-kowalczyk
-leonmsft
-Lepilleur
-Manandhar
-mbadolato
-Mehrain
-mgravell
-michaelniksa
-migrie
-mikegr
-mikemaccana
-miniksa
-niksa
-oising
-oldnewthing
-osgwiki
-paulcam
-pauldotknopf
-Pham
-Rincewind
-rprichard
-Schoonover
-Somuah
-sonph
-sonpham
-stakx
-Walisch
-Wirt
-Wojciech
-zadjii
-Zamor
-Zoey
-zorio
-Zverovich
+Anup

```

Comment:
Um, did you change the line endings from Unix to Windows?

(The spell checker seems to hate you, and I'm trying to figure out why. If that's the right problem, I'll write a commit for the next version.)

Reply:
Probably, since I opened it with Visual Studio to edit.  IIRC, there's a setting somewhere that enforces "checkout as Windows / checkin as Unixz" for line endings.  Most places I've worked that use git do that via a .gitconfig in the root of the repo, so it's never an issue.  I'd just figured you'd done the same.

I'll push again with Unix line endings.

Reply:
Looks good. Fwiw, I did end up writing the code to tolerate Windows line endings (I ran into a github repo that enforced Windows line endings which resulted in the tooling being very very unhappy, so I had to do the work).